src/lib/fs/fsUtils.ts

```diff
@@ -51,6 +68,24 @@ export function readFile(path: string): string {
     }
 }
 
+/**
+ * Reads the package.json file at path and returns javascript object.
+ * If path is not given, first package.json found in one of parent directories is used.
+ *
+ * @param path Optional path to package.json file
+ * @returns [[PackageJsonFile]] object
+ * @throws if path does not point to a file
+ */
+export function readPackageJson(path?: string): PackageJsonFile {
+  if (path == null) {
```

Comment:
Could you make this === undefined? ESLint is set to require === instead of == but I guess it didn't run?

Reply:
ESLint is allowing comparing with null using double-equals (allow-null-check option is set to true)(https://github.com/palantir/tslint/blob/master/src/configs/recommended.ts#L85)

Reply:
Oh I forgot I set that, thanks for reminding me. I might remove that setting later but for now its fine.